### PR TITLE
paragraphs

### PR DESCRIPTION
**Pull request by MHBauer**
there are a couple of suggestions here, I'll tag them individually below.

### CODE REVIEW

File: Lab 1/README.md [hunk 1]
```diff
@@ -2,18 +2,28 @@
 
 Learn how to push an image of an application to IBM Cloud Container Registry and deploy a basic application to a cluster.
 
-
-# 1. Push an image to IBM Cloud Container Registry
+# 0. Install Prerequisite CLIs and Provision a Kubernetes Cluster
```
Comment (MHBauer) [inline Lab 1/README.md:5]: I think we should separate out this prerequisites into it's own subsection, as it's a bit weird to say, "let's push an image" and then talk about something else first.

File: Lab 1/README.md [hunk 2]
```diff
@@ -2,18 +2,28 @@
...
 
 If you haven't already:
 1. Install the CLIs and Docker, as described in the "Prerequesite" section.  
 2. Provision a cluster: 
 
    ```bx cs cluster-create --name <name-of-cluster>```
 
-To push an image:
```
Comment (MHBauer) [inline Lab 1/README.md:14]: I moved the header down here

File: Lab 1/README.md [hunk 3]
```diff
@@ -2,18 +2,28 @@
...
+# 1. Push an image to IBM Cloud Container Registry
+
+To push an image, we first must have an image to push. We have
+prepared several `Dockerfile`s in this repository that will create the
+images. We will be running the images, and creating new ones, in the
+later labs. 
+
+This lab uses the Container Registry built in to IBM Cloud, but the
+image can be created and uploaded to any standard Docker registry to
+which your cluster has access.
```
Comment (MHBauer) [inline Lab 1/README.md:22]: This is some color commentary explaining why these steps exist, and explictily calling out IBM Cloud and it's standard compatibility.

File: Lab 1/README.md [hunk 4]
```diff
@@ -2,18 +2,28 @@
...
+
+1. Download a copy of this repository:
 
-1. Download from https://github.com/IBM/container-service-getting-started-wt
+    ```git clone https://github.com/IBM/container-service-getting-started-wt```
```
Comment (MHBauer) [inline Lab 1/README.md:26]: I don't know what "Download from" means. I'm assuming we want them to clone the repo, and assuming they know what that means.

File: Lab 1/README.md [hunk 5]
```diff
@@ -23,7 +33,7 @@ To push an image:
 
    ```bx login```
    
-   To specify an IBM Cloud region, include the API endpoint.
+   To specify an IBM Cloud region, include the API endpoint. <!-- what does this mean? can we add an example? -->
```
Comment (MHBauer) [inline Lab 1/README.md:36]: We should have an example of the flag or whatever necessary to choose the endpoint or remove this line.

As an FYI, html comments don't show up in the final output if we want to leave some kind of more detailed notes that don't show up:

screen shot 2018-02-07 at 3 25 38 pm

File: Lab 1/README.md [hunk 6]
```diff
@@ -45,11 +55,11 @@ To push an image:
 
 8. Now push that image up to IBM Cloud Container Registry: 
 
-   ```docker push registry.ng.bluemix.net/<namespace>/hello-world```
+   ```docker push registry.ng.bluemix.net/<my_namespace>/hello-world```
```
Comment (MHBauer) [inline Lab 1/README.md:58]: We never explain the angle bracket syntax, `<>` which I'm assuming is "the person who is going through the steps own choice of content". 

`namespace` did not match `my_namespace` above and I think it should.

**Review by jkomg — APPROVED** [review]
lgtm (and thank you)

**Pull request merged**